### ISSUE TITLE
RCall compatible with raspberry pi (julia arm version) ?

### ISSUE THREAD

**Issue by asasa137** [JuliaInterop/RCall.jl#385]
Hello,

I'm trying to use RCall on a raspberry pi. I installed succesfully R Stat and Julia package RCAL, however, when I do `using RCall` I get the following message :
```
using RCall

signal (11): Segmentation fault
in expression starting at REPL[4]:1
_ZL22isExtendedBUILD_VECTORPN4llvm6SDNodeERNS_12SelectionDAGEb at /home/pi/julia/julia-1.3.0/bin/../lib/julia/libLLVM-6.0.so (unknown line)
Allocations: 6194747 (Pool: 6194213; Big: 534); GC: 13
Segmentation fault
```

In lib/julia I have the specified file : 
```
ls lib/julia/libLL*
lib/julia/libLLVM-6.0.1.so  lib/julia/libLLVM-6.0.so  lib/julia/libLLVM.so
```

Do I have a missing file ? Has there been something wrong in my installation process ? Is RCall incompatible on ARM ?

I use julia V1.3.0 (32 bits ARM) on a raspberry pi v3.

Thanks,

Axel

**Comment by asasa137** [comment]
I tried Rcall with julia v1.4.1 and got the following error. It seems there are some incompatibility with Armv7l, with a dependency of rcall.
```
  using RCall
 [ Info: Precompiling RCall [6f49c342-dc21-5d91-9882-a32aef131414]
 ERROR: InitError: cfunction: closures are not supported on this platform
 Stacktrace:
  [1] safe_parseVector at /home/pi/.julia/packages/RCall/Qzssx/src/eval.jl:6 [inlined]
  [2] rparse_p(::Ptr{RCall.StrSxp}, ::Ptr{RCall.NilSxp}) at /home/pi/.julia/packages/RCall/Qzssx/src/eval.jl:49
  [3] rparse_p(::String, ::Ptr{RCall.NilSxp}) at /home/pi/.julia/packages/RCall/Qzssx/src/eval.jl:70 (repeats 2 times)
  [4] rparse(::String) at /home/pi/.julia/packages/RCall/Qzssx/src/eval.jl:74
  [5] set_hook(::String, ::RCall.RObject{RCall.LangSxp}) at /home/pi/.julia/packages/RCall/Qzssx/src/eventloop.jl:71
  [6] rgui_init() at /home/pi/.julia/packages/RCall/Qzssx/src/eventloop.jl:79
  [7] __init__() at /home/pi/.julia/packages/RCall/Qzssx/src/setup.jl:208
  [8] _include_from_serialized(::String, ::Array{Any,1}) at ./loading.jl:697
  [9] _require_from_serialized(::String) at ./loading.jl:748
  [10] _require(::Base.PkgId) at ./loading.jl:1039
  [11] require(::Base.PkgId) at ./loading.jl:927
  [12] require(::Module, ::Symbol) at ./loading.jl:922
 during initialization of module RCall```

**Comment by simonbyrne** [comment]
Fixed by #441
